sns.tf

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (C) 2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_sns_topic" "shuntingyard_ops_sns" {
+  name = "${local.instance_alias}-operational-events"
+}
+
+resource "aws_sns_topic_subscription" "sqs_hive_metastore_sns_subscription" {
```

Comment:
I think this should go in `sqs.tf` as you are subscribing the queue created to apiary events SNS topic. So, it will be easier for anyone reading the code to get the whole context at one place.

Reply:
Makes sense. Updated!